src/main/java/com/ukubuka/core/reader/UkubukaReader.java

```diff
@@ -81,4 +85,55 @@ public String readFileAsString(final SupportedSource source,
             throw new ReaderException(ex);
         }
     }
+
+    public String readXMLAsString(final SupportedSource source,
+                                   final String completeFileName)
+            throws ReaderException {
+        try {
+            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            InputStream in = new FileInputStream(source == SupportedSource.URL
+                    ? new File(
+                    new URL(completeFileName).toURI())
+                    : new File(completeFileName));
+            XMLStreamReader streamReader = inputFactory.createXMLStreamReader(in);
```

Comment:
Do we have to create `XMLStreamReader` instance every time?

Reply:
Fixed as I moved to bean!